lms/templates/verify_student/intro_step.underscore

```diff
@@ -46,7 +46,7 @@
 
             <div class="copy">
               <p>
-                <span class="copy-sub"><%- gettext( "A driver's license, passport, or other government-issued ID with your name and photo" ) %></span>
+                <span class="copy-sub"><%- gettext( "A driver's license, passport, or other government-issued ID with your name and photo." ) %></span>
```

Comment:
I don't think this needs a period - I suspect that stylistically, periods were omitted for these steps